orchestrator/dockerdir/entrypoint.sh

```diff
@@ -11,25 +11,28 @@ fi
 if [ -n "$KUBERNETES_SERVICE_HOST" ]; then
     PATH_ORC_CONF_FILE='/etc/orchestrator'
     jq -M ". + {
-                HTTPAdvertise:\"http://$HOSTNAME-svc:80\",
-                RaftAdvertise:\"$HOSTNAME-svc\",
-                RaftBind:\"$HOSTNAME\",
+                HTTPAdvertise:\"http://$HOSTNAME.$ORC_SERVICE:80\",
+                RaftAdvertise:\"$HOSTNAME.$ORC_SERVICE\",
+                RaftBind:\"$HOSTNAME.$ORC_SERVICE\",
                 MySQLTopologySSLPrivateKeyFile:\"/etc/orchestrator/ssl/tls.key\",
                 MySQLTopologySSLCertFile:\"/etc/orchestrator/ssl/tls.crt\",
                 MySQLTopologySSLCAFile:\"/etc/orchestrator/ssl/ca.crt\",
-                RaftNodes:[\"$HOSTNAME-svc\"]
+                RaftNodes:[\"$HOSTNAME.$ORC_SERVICE\"]
           }" "${PATH_ORC_CONF_FILE}/orchestrator.conf.json" 1<>"${PATH_ORC_CONF_FILE}/orchestrator.conf.json"
 
     { set +x; } 2>/dev/null
     PATH_TO_SECRET='/etc/orchestrator/orchestrator-users-secret'
     if [ -f "$PATH_TO_SECRET/TOPOLOGY_PASSWORD" ]; then
         TOPOLOGY_PASSWORD=$(/bin/cat $PATH_TO_SECRET/TOPOLOGY_PASSWORD)
     fi
-    if [ -f "$PATH_TO_SECRET/TOPOLOGY_UASER" ]; then
+    if [ -f "$PATH_TO_SECRET/TOPOLOGY_USER" ]; then
         TOPOLOGY_USER=$(/bin/cat $PATH_TO_SECRET/TOPOLOGY_USER)
     fi
-    sed -r "s|^[#]?user=.*$|user=${TOPOLOGY_USER:-$ORC_TOPOLOGY_USER}|" "${PATH_ORC_CONF_FILE}/orc-topology.cnf" 1<>"${PATH_ORC_CONF_FILE}/orc-topology.cnf"
-    sed -r "s|^[#]?password=.*$|password=${TOPOLOGY_PASSWORD-$ORC_TOPOLOGY_PASSWORD}|" "${PATH_ORC_CONF_FILE}/orc-topology.cnf" 1<>"${PATH_ORC_CONF_FILE}/orc-topology.cnf"
```

Comment:
I do not see any issues with old approach :( Everything works for me.

Reply:
please run: `docker run --rm -e KUBERNETES_SERVICE_HOST=host -e ORC_TOPOLOGY_PASSWORD=mypass perconalab/percona-server-mysql-operator:main-orchestrator`

Reply:
I see the issue. Thank you!